project/mima-filters/10.0.0.excludes

```diff
@@ -0,0 +1,7 @@
+# Change in internal class
```

Comment:
I think it'd be good to explain with a comment here that this file means "changes from 10.0.0" as it may be confusing for people if they should add things to 10.0.1 or 10.0.0 sometimes?